src/main/resources/static/api.yml

```diff
@@ -218,7 +221,7 @@ components:
       properties:
         name:
           type: string
-        acronym:
+        alternateName:
```

Comment:
If this is 'alternateName' then the 'acronym' column in the Courses tab of the spreadsheet should also be called 'alternateName'.